main.go

```diff
@@ -30,6 +30,33 @@ type idList struct {
 	IdList []int64 `json:"ids" binding:"required"`
 }
 
+func updateFlag(value string, statusOk bool, varName string) {
+	if statusOk {
+		_, err := strconv.ParseBool(value)
+		if err == nil {
+			os.Setenv(varName, value)
+		} else {
+			log.Println("Invalid config value for ", varName)
+		}
+	}
+}
+
+func config(c *gin.Context) {
+	analytics, analyticsOk := c.GetQuery("analytics")
+	faves, favesOk := c.GetQuery("faves")
+	login, loginOk := c.GetQuery("login")
+
+	updateFlag(analytics, analyticsOk, "STORE_ANALYTICS")
+	updateFlag(faves, favesOk, "CALL_FAVES")
+	updateFlag(login, loginOk, "CALL_LOGIN")
+
+	c.HTML(http.StatusOK, "index.tmpl.html", gin.H{
+		"analytics_status": os.Getenv("STORE_ANALYTICS"),
+		"faves_status":     os.Getenv("CALL_FAVES"),
```

Comment:
We don't need those, I find it mildly confusing to even parse them. Because as far as I know, we're not calling faves or login? Or are we?

Reply:
I am not sure and I am double-checking with frank on Slack